modules/runner/src/integrationTest/java/org/apache/ignite/internal/table/ItTableScanTest.java

```diff
@@ -178,9 +181,8 @@ public void testInsertDuringScan() throws Exception {
 
         subscription.request(1);
 
-        waitForCondition(() -> !scannedRows.isEmpty(), 10_000);
+        assertTrue(waitForCondition(() -> !scannedRows.isEmpty(), 10_000));
 
-        assertEquals(1, scannedRows.size());
```

Comment:
The change is not equivalent: earlier it checked that there is exactly 1 element, now it will allow any non-zero number of elements